webserver/__init__.py

```diff
@@ -20,24 +31,30 @@ def create_app_with_configuration(config_path=None):
     )
 
     # Configuration
-    root_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..")
-    app.config.from_pyfile(os.path.join(
-        root_path,
-        'default_config.py'
-    ))
-    app.config.from_pyfile(os.path.join(
-        root_path,
-        'custom_config.py'
-    ), silent=True)
+    config_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "config.py")
+    print(config_file)
```

Comment:
Not sure if we need this now since there is only one config file.

Reply:
For the MeB setup, I actually generate the consul config to consul_config.py and refuse to even look at config.py should it miraculously appear in the repo. (I've also added config.py to .dockerignore) This makes it extra difficult to get the wrong config file... Perhaps even impossible.